.github/workflows/cibuild.yml

```diff
@@ -3,6 +3,8 @@ name: CI
 on:
   push:
   pull_request:
+  schedule:
```

Comment:
I've just noticed this section.  What is the intended effect of this `cron`?  I'm new to this syntax so it looks like this scheduling is independent of the other triggers (so the CI would run every 3 hours all the time), but I could also imagine it somehow chaining with the push and pull_request to bin the CI runs once those triggers happen.  Are there reasons to run other than when a push or pull_request trigger happens?

Reply:
I think that spec tells it to run **daily** at 3 AM (UTC). If this is a low-traffic package, weekly might work too.

Cron job is to catch upstream changes that might break your code. So I don't think you should get rid of it. Most of the time, it would pass, but for that rare case a new upstream release breaks your code, you would want to know even when there is no push to `master`. Most common example is that a new release finally removed that deprecated API and you have been ignoring the deprecation warning; or it removed/changed a hidden function that you have been using. Hope that clarifies the matter.

Reply:
The nightly build cron runs at 3:00UTC in the master.

Reply:
Yes, sorry, I don't know why I read that as every 3 hours.  Daily seems OK to me.  It's nice to see those upstream impacts sooner rather than later.

Reply:
Yeah, I'm also nervous about build breakages due to outside things, even outside astropy, as sometimes it's a while between PRs.